uw-frame-components/css/buckyless/general.less

```diff
@@ -5,7 +5,7 @@ body {
 }
 a {
   transition: @link-hover-transition;
-  &:not(.md-button), &:not(.btn) {
```

Comment:
Interesting. The result is the same, right? In the absence of `:not` this would break some styles, but because it's a `:not`, does it make any difference whether it's "and" or "or"?